Makefile.in

```diff
@@ -22,22 +22,29 @@ all:
 	$(GOBUILD) -o accept acceptance_tests/accept.go
 	$(GOBUILD) -o masapi mas/api/api.go
 
+concurrent: concurrent.c
+	$(CC) -Wall -O2 $< -o $@
+
+concurrent.c:
+	wget --quiet https://github.com/seanpringle/concurrent/raw/master/concurrent.c
```

Comment:
Worth noting a raw version from a specific hash could be safer: https://raw.githubusercontent.com/seanpringle/concurrent/e6ff2d98fb7f4cebf297b1725fee9a9e3d1cc164/concurrent.c

Reply:
Just to be clear: this is not a request, nor necessarily a blocker. But if this script is eventually pulled in by the deployment process I think others will expect a specific version.